### ISSUE TITLE
A problem with textAlign documentation

### ISSUE THREAD

**Issue by JimBrink** [processing/p5.js#5142]
#### Most appropriate sub-area of p5.js?
- [x] Other (specify if possible)
 documentation
#### Details about the bug:
The documentation page https://p5js.org/reference/#/p5/textAlign seems to have 2 problems:
1.  Nowhere does it say the default vertical alignment is "BASELINE" if the
  textAlign call does not specify the vertical alignment.
2.  The second example on that page shows the BASELINE alignment as being the same as TOP.  (The may be problem with the p5,js source file the documentation uses.)

- p5.js version :  p5.js v1.2.0 Dec
- Web browser and version: <Firefox vs 87.0, opera (Same results for both the code below and on the documentation page.
- Operating System:  Windows 10
- Steps to reproduce this:   Compare the results for  "BASELINE" shown on the document page with those in this code.

```js
function setup() {

  createCanvas(400, 100);
  background("GRAY");
  fill(255);
}

function draw() {

  line(0,30,400,30);
  text('default', 30, 30);
  textAlign(CENTER, TOP);
  text("TOP", 110, 30);
  textAlign(CENTER, BASELINE);
  text ("BASELINE", 190, 30);
  textAlign(CENTER, BOTTOM);
  text ("BOTTOM", 270, 30);
  textAlign(CENTER, CENTER);
  text ("CENTER", 350, 30);
  noLoop();
}  
```

**Comment by limzykenneth** [comment]
@two-ticks If you are still interested, do have go ahead. I think there are some weird behaviour going on so do have a closer look while you are working on this.

**Comment by dhowe** [comment]
@two-ticks please also compare to processing: https://processing.org/reference/textAlign_.html

**Comment by Qianqianye** [comment]
Thanks @two-ticks. I just assigned you on this.

**Comment by Qianqianye** [comment]
Closing this since it's resolved in #6854
